lib/install/action/fetch.js

```diff
@@ -24,6 +24,6 @@ module.exports = function (top, buildpath, pkg, log, next) {
       name = pkg.package._requested.raw
   }
   log.silly('fetch', packageId(pkg))
-  cache.add(name, version, top, false, next)
+  cache.add(name, version, pkg.parent.path, false, next)
```

Comment:
this commit implies this is a refactor -- why is `pkg.parent.path` expected to always be the same as `top`? Was this a bug before?